src/main/java/com/minecolonies/coremod/blocks/ModBlocks.java

```diff
@@ -71,6 +73,9 @@ public static void init(final IForgeRegistry<Block> registry)
         blockHutWareHouse = (BlockHutWareHouse) new BlockHutWareHouse().registerBlock(registry);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Synchronize this lazy initialization of 'blockHutWareHouse' [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS2444)